### DISCUSSION TITLE
Is the a way to set plain text encode?

### DISCUSSION THREAD

**Discussion by impartialjust** [PHPMailer/PHPMailer#2497] [Q&A] [answered]
The default is us-ascii or iso-8859-1, here is a exapmle for iso-8859-1:
```
--b1_OMTWg48WpfRl7yHrGVqlB5Q9LfadsfsMfWLOytzs3iGk
Content-Type: text/plain; charset=iso-8859-1
Content-Transfer-Encoding: 8bit

è¡£æœæ™¾äº†å—:å³å¿—å“

--b1_OMTWg48WpfRl7yHrGVqlB5Q9LfadsfsMfWLOytzs3iGk
```
So is that possible to change charset?
Thanks.

**Comment by Synchro** [comment, marked as answer]
Yes, change [the `CharSet` property](http://phpmailer.github.io/PHPMailer/classes/PHPMailer-PHPMailer-PHPMailer.html#property_CharSet):

    $mail->CharSet = 'utf-8';

There are [defined constants for common charsets](http://phpmailer.github.io/PHPMailer/classes/PHPMailer-PHPMailer-PHPMailer.html#constant_CHARSET_ASCII), so you can also say:

    $mail->CharSet = PHPMailer::CHARSET_UTF8;
